dali/operators/image/peek_shape/peek_shape.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/image/peek_shape/peek_shape.h"
+
+namespace dali {
+
+DALI_SCHEMA(PeekShape)
```

Comment:
This is image specific, so I think the name should reflect it. Something like `ImagePeekShape`?

Reply:
Done